### PR TITLE
R4 Person Read and Search Documentation Added

### PR DESCRIPTION
**Pull request by sonikrish**
Description
----
The R4 documentation for Person Read and Search

Validation
---------
Screenshot 2020-06-23 at 1 25 21 AM
Screenshot 2020-06-23 at 12 55 49 AM
Screenshot 2020-06-23 at 12 55 45 AM

Screenshot 2020-06-23 at 12 55 42 AM
Screenshot 2020-06-23 at 12 55 38 AM
Screenshot 2020-06-23 at 1 21 42 AM

### CODE REVIEW

File: content/millennium/r4/individuals/person.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Parameters
+
+ Name                 | Required?                                    | Type       | Description
+----------------------|----------------------------------------------|------------|--------------------------------------------------------------------------
+ `_id`                | No, if the identifier is populated           | [`token`]  | The logical resource id associated with the resource.
```
Comment (ebaranowski) [inline content/millennium/r4/individuals/person.md:43]: No, if the identifier is populated -> This, or any other required search parameter

File: content/millennium/r4/individuals/person.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+ `identifier`         | Yes, or `_id`                                | [`token`]  | The person identifier.  Example: `urn:oid:2.16.840.1.113883.3.13.6|01022228`
```
Comment (ebaranowski) [inline content/millennium/r4/individuals/person.md:44]: Yes, or `_id` -> This or `_id`

File: lib/resources/example_json/r4_examples_person.rb [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+  
+      R4_COMBINED_PERSON_ENTRY ||= {
+        "resourceType": "Person",
+        "id": "971923",
+        "meta": {
+            "versionId": "0"
+        },
+        "text": {
+            "status": "generated",
+            "div": "&lt;div xmlns=\"http://www.w3.org/1999/xhtml\"&gt&lt;p&gt&lt;b&gtPerson&lt;/b&gt&lt;/p&gt&lt;p&gt&lt;b&gtStatus&lt;/b&gt: Inactive&lt;/p&gt&lt;p&gt&lt;b&gtLink&lt;/b&gt: Person/933924&lt;/p&gt&lt;/div&gt"
```
Comment (ebaranowski) [inline lib/resources/example_json/r4_examples_person.rb:12]: All of the end tags in all of these examples are not rendering correctly

File: content/millennium/r4/individuals/person.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
+---
```
Comment (ebaranowski) [inline content/millennium/r4/individuals/person.md:1]: Need to add the disclaimer to the examples: https://github.com/cerner/fhir.cerner.com/blame/master/content/millennium/r4/general-clinical/condition.md#L108

File: lib/resources/example_json/r4_examples_person.rb [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+        "link": [
+            {
+                "relation": "self",
+                "url": "https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Person?identifier=urn%3Aoid%3A1.1.1.1.1.1%7C10000360"
+            }
+        ],
+        "entry": [
+            {
+                "fullUrl": "https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Person/1316024",
+                "resource": {
```
Comment (ebaranowski) [inline lib/resources/example_json/r4_examples_person.rb:170]: Helps reduce the redundancy: https://github.com/cerner/fhir.cerner.com/blob/master/lib/resources/example_json/r4_examples_allergy_intolerance.rb#L135

File: content/millennium/r4/individuals/person.md (hunk 1 above)
Comment (sonikrish) [reply content/millennium/r4/individuals/person.md]: Addressed in commit 6b67bcc19cfed472c7ec020f38e7d45ab0067dcc

File: content/millennium/r4/individuals/person.md (hunk 2 above)
Comment (sonikrish) [reply content/millennium/r4/individuals/person.md]: Changed in commit 6b67bcc19cfed472c7ec020f38e7d45ab0067dcc

File: lib/resources/example_json/r4_examples_person.rb (hunk 3 above)
Comment (sonikrish) [reply lib/resources/example_json/r4_examples_person.rb]: Corrected in commit 6b67bcc19cfed472c7ec020f38e7d45ab0067dcc

File: content/millennium/r4/individuals/person.md (hunk 4 above)
Comment (sonikrish) [reply content/millennium/r4/individuals/person.md:1]: added in commit 6b67bcc19cfed472c7ec020f38e7d45ab0067dcc

File: lib/resources/example_json/r4_examples_person.rb (hunk 5 above)
Comment (sonikrish) [reply lib/resources/example_json/r4_examples_person.rb]: Added in commit 6b67bcc19cfed472c7ec020f38e7d45ab0067dcc

File: lib/resources/example_json/r4_examples_person.rb [hunk 6]
```diff
@@ -1,323 +1,173 @@
...
+      "meta": {
+        "versionId": "0"
+      },
+      "text": {
+        "status": "generated",
+          "div": "&lt;div xmlns=\"http://www.w3.org/1999/xhtml\"&gt;&lt;p&gt;&lt;b&gt;Person&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: Inactive&lt;/p&gt;&lt;p&gt;&lt;b&gt;Link&lt;/b&gt;: Person/933924&lt;/p&gt;&lt;/div&gt;"
+      },
+      "active": false,
+      "link": [
+      {
```
Comment (ebaranowski) [inline lib/resources/example_json/r4_examples_person.rb:16]: These tab alignments are off

File: content/millennium/r4/individuals/person.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+The following fields are returned if valued:
+
+* [Person id](http://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Identifier (Cerner alias Federated Person Principal)](http://hl7.org/fhir/R4/person-definitions.html#Person.identifier){:target="_blank"}
+* [Name](http://hl7.org/fhir/R4/person-definitions.html#Person.name){:target="_blank"}
+* [Telecom](http://hl7.org/fhir/R4/person-definitions.html#Person.telecom){:target="_blank"}
+* [Gender](http://hl7.org/fhir/R4/person-definitions.html#Person.gender){:target="_blank"}
+* [Date of birth](http://hl7.org/fhir/R4/person-definitions.html#Person.birthDate){:target="_blank"}
+* [Address](http://hl7.org/fhir/R4/person-definitions.html#Person.address){:target="_blank"}
+* [Active](http://hl7.org/fhir/R4/person-definitions.html#Person.active){:target="_blank"}
```
Comment (nishanthnamana) [inline content/millennium/r4/individuals/person.md:23]: We should we secure links here.

File: lib/resources/r4/person.yaml [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+- name: identifier
+  type: List of Identifier
+  description: The identifier for the person.
+  action: terminology
+  children:
+
+  - name: type
+    type: CodeableConcept
+    description: The type for the identifier that can be used to determine which identifier to use for a specific purpose.
+    url: http://hl7.org/fhir/r4/datatypes-definitions.html#Identifier.type
```
Comment (nishanthnamana) [inline lib/resources/r4/person.yaml:14]: Secure link here as well. Use 'https'

Comment (sonikrish) [reply lib/resources/r4/person.yaml]: Changed in commit 8f913dce83505df0c6e3815d58fb4cf8367d8b44

File: content/millennium/r4/individuals/person.md (hunk 7 above)
Comment (sonikrish) [reply content/millennium/r4/individuals/person.md]: Changed in commit 8f913dce83505df0c6e3815d58fb4cf8367d8b44

Comment (nishanthnamana) [reply content/millennium/r4/individuals/person.md]: +1

File: lib/resources/r4/person.yaml [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+  - name: type
+    type: CodeableConcept
+    description: The type for the identifier that can be used to determine which identifier to use for a specific purpose.
+    url: https://hl7.org/fhir/r4/datatypes-definitions.html#Identifier.type
+    action: terminology
+    binding:
+      description: The type used to determine which identifier to use for a specific purpose.
+      terminology:
+      - display: Identifier Type Codes
```
Comment (ebaranowski) [inline lib/resources/r4/person.yaml:19]: This can be removed

Comment (sonikrish) [reply lib/resources/r4/person.yaml]: removed in commit f6e7999c7e7fb613eeacc38ecfe1f445a4e9480d

**Comment by MaxPhilips** [comment]
verification on the live site

Screen Shot 2020-06-23 at 5 07 27 PM

**Pull request merged**